ddtrace/contrib/aiohttp_jinja2/patch.py

```diff
@@ -15,6 +17,10 @@
 )
 
 
+def get_version():
+    return getattr(aiohttp_jinja2, "__version__", "")
```

Comment:
nit: we can probably add typing to all the get version methods. This isn't blocking

```suggestion
    # type: () -> str
    return getattr(aiohttp_jinja2, "__version__", "")
```

Reply:
Done.